### DISCUSSION TITLE
An exception occurred while initializing CoSimulationSlave

### DISCUSSION THREAD

**Discussion by wangtianying1111** [NTNU-IHB/FMI4j#146] [Q&A] [answered]
When the program runs to “CoSimulationSlave simulationSlave = coSimulationFmu.newInstance();”
An exception was thrown

[FMI native bridge] Fatal: Failed to load library '/tmp/fmi4j_Model_SpringMassDamping_Original_FMU_RealTime_LINUX_X64169493463264149197/binaries/linux64/Model_SpringMassDamping_Original_FMU.so', error: /tmp/fmi4j_Model_SpringMassDamping_Original_FMU_RealTime_LINUX_X64169493463264149197/binaries/linux64/Model_SpringMassDamping_Original_FMU.so: cannot open shared object file: No such file or directory
fmi4j_Model_SpringMassDamping_Original_FMU_RealTime_LINUX is my fmu file name
/
I checked this directory，no file in. so format

![image](https://github.com/NTNU-IHB/FMI4j/assets/151618259/30f6b420-d547-4fc5-a76b-e4d86e2cb08c)

**Comment by markaren** [comment, marked as answer]
If there is no shared library in your FMU for the given platform, FMI4j cannot load the model

**Reply by wangtianying1111** [reply]
Platform here refers to the operating system or jvm? Looking forward to your reply, thank you.

**Reply by wangtianying1111** [reply]
[Model_SpringMassDamping_Realtime_FMU_Linux_x64_Binary.fmu.zip](https://github.com/NTNU-IHB/FMI4j/files/13424027/Model_SpringMassDamping_Realtime_FMU_Linux_x64_Binary.fmu.zip)
this is my FMU

**Reply by markaren** [reply]
Operating System

**Reply by wangtianying1111** [reply]
thanks very much for your help
